lib/parser.dart

```diff
@@ -2217,8 +2215,10 @@ class _Parser {
   }
 
   // TODO(terry): Need to handle auto.
-  num marginValue(var exprTerm) {
-    if (exprTerm is UnitTerm || exprTerm is NumberTerm) {
+  num marginValue(Expression exprTerm) {
+    if (exprTerm is UnitTerm) {
+      return exprTerm.value as num;
```

Comment:
No need for `as num` here as `UnitTerm.value` has type `int` (which extends `num`).

Reply:
Unfortunately not :(